woocommerce/woocommerce
# [CYS] Avoid the intro cards to get smaller on large screens

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PRs improves the design of the CYS Intro page. It avoids the intro cards to get smaller at bigger screens.

Closes https://github.com/woocommerce/woocommerce/issues/48947



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.
2. Click the `Start designing` button and wait for the assembler to load.
3. Go again to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.
4. Using the dev tools on your browser, make the screen wider and confirm the cards at the bottom do not get smaller and the max width they reach is `500px`.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Improve the designs of the Intro page bottom cards.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
